src/query/service/src/pipelines/processors/transforms/window/partition_by/transform_window_partition_sort.rs

```diff
@@ -26,7 +26,8 @@ use databend_common_pipeline_transforms::processors::sort_merge;
 use databend_common_pipeline_transforms::processors::BlockMetaTransform;
 use databend_common_pipeline_transforms::processors::BlockMetaTransformer;
 
-use crate::pipelines::processors::transforms::WindowPartitionMeta;
+use super::Partitioned;
+use super::WindowPartitionMeta;
 
 pub struct TransformWindowPartitionSort {
```

Comment:
This PR should be merged after A is merged, because A refactored the window spill.